torch/_inductor/codegen/cpp_wrapper_cuda.py

```diff
@@ -183,6 +183,9 @@ def generate_kernel_call(
                 name, call_args, grid, device_index, cuda, triton, arg_types
             )
 
+        device_index, call_args = self.prepare_triton_kernel_call(
```

Comment:
This is not triton specific right? Maybe just call it 'prepare_kernel_call'

Reply:
Actually it is Triton specific, at least by looking at all its call sites.